client/reader/stream/reader-tag-sidebar/index.jsx

```diff
@@ -33,11 +35,24 @@ const ReaderTagSidebar = ( { tag, showFollow } ) => {
 	} );
 	const isFollowing = useSelector( ( state ) => getReaderTagBySlug( state, tag )?.isFollowing );
 	const isLoggedIn = useSelector( ( state ) => isUserLoggedIn( state ) );
+	const { data: prompts } = useBloggingPrompts( 1, 10 );
```

Comment:
Haven't debugged too much but this isn't working for me. The `prompts` here is undefined, so nothing shows up in sidebar.

Reply:
It seems the first argument to `useBloggingPrompts` is supposed to be the siteId.

Reply:
oops, forgot to push a commit last week, this should be fixed now.